tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/partition/PartitionDesc.java

```diff
@@ -50,15 +73,21 @@ public PartitionDesc(CatalogProtos.PartitionDescProto proto) {
     if(proto.hasPartitionName()) {
       this.partitionName = proto.getPartitionName();
     }
-    this.ordinalPosition = proto.getOrdinalPosition();
-    if(proto.hasPartitionValue()) {
-      this.partitionValue = proto.getPartitionValue();
+
+    this.partitionKeys = new ArrayList<PartitionKey>();
+    for(CatalogProtos.PartitionKeyProto keyProto : proto.getPartitionKeysList()) {
+      PartitionKey partitionKey = new PartitionKey(keyProto);
+      this.partitionKeys.add(partitionKey);
     }
+
     if(proto.hasPath()) {
       this.path = proto.getPath();
     }
   }
 
+  public String getPartitionName() { return partitionName; }
+  public void setPartitionName(String partitionName) { this.partitionName = partitionName; }
```

Comment:
Is this function different with the below setName() function?
In addition, please move brackets according to our coding convention.